#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_installer.dpatch by  <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Install applications in /usr/local/lib instead of /usr/lib

@DPATCH@

=== xulrunner/setup/nsXULAppInstall.js
==================================================================
--- xulrunner/xulrunner/setup/nsXULAppInstall.js	(revision 65)
+++ xulrunner/xulrunner/setup/nsXULAppInstall.js	(local)
@@ -246,7 +246,7 @@
 #else
       aDirectory = Components.classes["@mozilla.org/file/local;1"].
         createInstance(nsILocalFile);
-      aDirectory.initWithPath("/usr/lib");
+      aDirectory.initWithPath("/usr/local/lib");
       if (vendor)
         aDirectory.append(vendor.toLowerCase());
 #endif
